packages/apex-ls/src/communication/MessageTransports.ts

```diff
@@ -96,7 +102,7 @@ export class SelfMessageTransport implements MessageTransport {
   }
 
   onError(handler: (error: Error) => void): { dispose(): void } {
-    const errorHandler = (event: ErrorEvent) => {
+    const errorHandler = (event: any) => {
```

Comment:
Why the type loosening?

Reply:
I've tested adding these back in after importing webworker and the ErrorEvent works as well!